src/test/java/org/assertj/core/api/string_/StringAssert_isEqualToNormalizingUnicode_Test.java

```diff
@@ -0,0 +1,41 @@
+package org.assertj.core.api.string_;
+
+import org.assertj.core.api.StringAssert;
+import org.assertj.core.api.StringAssertBaseTest;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
+import static org.mockito.Mockito.verify;
+
+public class StringAssert_isEqualToNormalizingUnicode_Test extends StringAssertBaseTest {
+
+  @Override
+  protected StringAssert invoke_api_method() {
+    return assertions.isEqualToNormalizingUnicode("\u0041\u0308");
+  }
+
+  @Override
+  protected void verify_internal_effects() {
+    verify(objects).assertEqual(getInfo(assertions), getActual(assertions), "\u00C4");
+
+  }
+
+  @Test
+  public void should_pass_equals_normalized_String_when_compared_with_normalized_String(){
```

Comment:
this will be moved and adapted to `Strings_assertEqualsNormalizingUnicode_Test`, a good reference to write it is `Strings_assertEqualsNormalizingPunctuationAndWhitespace_Test`